english/count-the-marshmellows/eat-all-the-marshmellows/get-breakfast/get-breakfast.md

```diff
@@ -1,7 +1,14 @@
-HECK YES. You found the best ending. (Breakfast is always the best ending.)
 Your favorite diner is busy but not crowded. Somehow, Sam always knows when
 you're coming, and she puts a hot plate of blueberry pancakes on your table as you sit
 down. Your buddy orders a cup of coffee while you cut the pancakes into
 shareable slices and pour syrup into the center. Today's already looking up.
 
 You begin to eat your breakfast, but everything on your plate tastes like marshmallow.
+
+As the waitress places the check on your table, she says, "There are no coincidences." Her hand moves to reveal a tiny marshmallow. "Think about it," she urges before retreating to the kitchen.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.